google/cloud/forseti/notifier/notifiers/email_violations.py

```diff
@@ -161,16 +184,31 @@ def _send(self, **kwargs):
         content = notification_map['content']
         attachment = notification_map['attachment']
 
-        try:
-            self.mail_util.send(
-                email_sender=self.notification_config['sender'],
-                email_recipient=self.notification_config['recipient'],
-                email_subject=subject,
-                email_content=content,
-                content_type='text/html',
-                attachment=attachment)
-        except util_errors.EmailSendError:
-            LOGGER.warn('Unable to send Violations email')
+        if self.notifier_config.get('email_connector'):
+            try:
+                self.connector.send(
```

Comment:
Can you call `self.connector.send()` once?
```
if self.notifier_config.get('email_connector'):
    sender=self.notifier_config.get('email_connector').get('sender')
    recipient=self.notifier_config.get('email_connector').get('recipient')
else:
    sender = self.notification_config['sender']
    recipient = self.notification_config['recipient']

self.connector.send(
    blah,
    blah,
    blah
)
```

Reply:
Done